package.json

```diff
@@ -30,40 +30,31 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "request": "~2.69.0",
-    "underscore": "1.8.3",
-    "cheerio": "0.20.0",
-    "xmldom": "0.1.22",
-    "logops": "1.0.0-alpha.7",
+    "async": "2.6.1",
+    "request": "~2.88.0",
+    "underscore": "1.9.1",
+    "cheerio": "1.0.0-rc.2",
+    "xmldom": "0.1.27",
+    "logops": "2.0.0",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "lwm2m-node-lib": "git://github.com/dcalvoalonso/lwm2m-node-lib.git#task/updateDependencies"
```

Comment:
I understand we should retain this PR until PR https://github.com/telefonicaid/lwm2m-node-lib/pull/136 gets merged (and this line reverted).

Reply:
You're right, we have to wait! ;)

Reply:
I understand this has been changed back in the current status of the branch. NTC